zdm/grid.py

```diff
@@ -379,6 +389,11 @@ def GenMCSample(self,N,Poisson=False):
             which w bin it generates.
         
         """
+        # Boost?
```

Comment:
Can you adda comment about precisely what "boost" means? Is this the cpp code?

Reply:
IGNORE the above - sadly, the webpage is not letting me delete the comment for some inane reason...